src/events/cd-user-ticket-list-item.vue

```diff
@@ -24,7 +24,7 @@
         <a :href="event.eventbriteUrl | cdUrlFormatter" target="_blank" class="btn btn-lg btn-primary cd-user-ticket-list-item__view">{{ $t('See Details and Book') }}</a>
       </div>
       <div v-if="hasApplications" class="cd-user-ticket-list-item__view-applications">
-        <h4 class="cd-user-ticket-list-item__view-application-title"><i class="fa fa-ticket"></i>{{ $t('Tickets') }}</h4>
+        <h4 class="cd-user-ticket-list-item__view-application-title"><i class="fa fa-ticket"></i>{{ $tc('Tickets') }}</h4>
```

Comment:
Using $tc is useless without

Reply:
true, will remove tc and default to plurals